addons/storyshots/storyshots-core/tsconfig.json

```diff
@@ -6,6 +6,7 @@
   "compilerOptions": {
     "rootDir": "./src",
     "experimentalDecorators": true,
-    "declaration": true
+    "declaration": true,
+    "skipLibCheck": true
```

Comment:
Storyshots is old, and contains code for multiple frameworks, it gets 'confused' about JSX and frameworks fight over it. Let's skip this for now.